components/switch/switch-mixin.js

```diff
@@ -21,8 +21,7 @@ export const SwitchMixin = superclass => class extends FocusMixin(RtlMixin(Focus
 			 */
 			on: { type: Boolean, reflect: true },
 			/**
-			 * REQUIRED: The text that is displayed for the switch label.
-			 * @type {string}
+			 * @ignore - Need to add documentation in each component that uses this mixin.
```

Comment:
Does `Need to add documentation in each component that uses this mixin` get added to custom-elements.json at all or does the `@ignore` get priority?

Reply:
So to directly answer your question, it seems `ignore` gets priority and I can get away with `Need to add documentation in each component that uses this mixin` as an extra comment for future developers.

Reply:
Great, yeah Daylight pulls from `attributes` and not `properties`